src/utils/timeSeriesResultsByOption.js

```diff
@@ -76,6 +80,7 @@ export function getMonthly ({
         longerOption.mortgageRate,
         investmentRate,
         inflationRate,
+        shorterOptionPMT - longerOptionPMT,
```

Comment:
for the 'longetOption', the `investmentPMT` is always equal to the difference between `shorterOptionPMT` and `longerOptionPMT`